src/include/neorados/RADOS_Decodable.hpp

```diff
@@ -104,4 +104,10 @@ struct hash<::neorados::Entry> {
 };
 }
 
+#include <fmt/core.h>
```

Comment:
i'd prefer moving the `#include`s to the very beginning of this file. so it looks like

```c++
/// ...
#include <vector>

#if FMT_VERSION >= 90000
#include <fmt/core.h>
#include <fmt/ostream.h>
#endif

// ...
#if FMT_VERSION >= 90000
#include <fmt/ostream.h>
template <> struct fmt::formatter<neorados::Entry> : ostream_formatter {};
#endif
```

note that i removed the closing `// FMT_VERSION >= 90000` comment. because i think the `#if` block is pretty small, it's probably not very helpful to have this comment.

Reply:
Done!